### PR TITLE
Floor changing implemented

### PR DESCRIPTION
**Pull request by pjpolley**

### CODE REVIEW

**Comment by iancj97** [comment]
Hold up on this one sec, making changes.

**Comment by iancj97** [comment]
done. No changes needed, floors load as expected.

File: src/Controller/User/UserMapViewController.java [hunk 1]
```diff
@@ -222,6 +224,52 @@ public void addToMap(MapNode n)
     }*/
 
     @FXML
+    private void clickedDownArrow(){
+        int desiredFloor = model.getCurrentFloor().getFloorNumber() - 1;
+        boolean foundFloor = false;
+        for(Building b : model.getHospital().getBuildings()) {
+            if(b.getFloors().contains(model.getCurrentFloor())) {
+                for(Floor f : b.getFloors()) {
```
Comment (benhylak) [inline src/Controller/User/UserMapViewController.java]: Should be b.getFloor(int)

File: src/Controller/User/UserMapViewController.java [hunk 2]
```diff
@@ -222,6 +224,52 @@ public void addToMap(MapNode n)
...
+    }
+
+    @FXML
+    private void clickedUpArrow(){
+        int desiredFloor = model.getCurrentFloor().getFloorNumber() + 1;
+        boolean foundFloor = false;
+        for(Building b : model.getHospital().getBuildings()) {
+            if(b.getFloors().contains(model.getCurrentFloor())) {
+                for(Floor f : b.getFloors()) {
+                    if(f.getFloorNumber() == desiredFloor){
```
Comment (benhylak) [inline src/Controller/User/UserMapViewController.java]: here too

File: src/Controller/User/UserMapViewController.java [hunk 3]
```diff
@@ -314,6 +362,15 @@ private void initialize() throws Exception {
 
         panel.setVisible(false);
         directionPaneView();
+
+        for( Building b : model.getHospital().getBuildings()) {
+            if(b.getFloors().contains(model.getCurrentFloor())) {
+                for(Floor f : b.getFloors()) {
+                    dropDown.getItems().add(Integer.toString(f.getFloorNumber()));
```
Comment (benhylak) [inline src/Controller/User/UserMapViewController.java:437]: Will be super weird for multiple buildings.... Consider asking building names to the string or something?

File: src/View/User/UserMapView.fxml [hunk 4]
```diff
@@ -25,6 +25,7 @@
 <?import javafx.scene.control.TableView?>
 <?import javafx.scene.control.TableColumn?>
 
+<?import javafx.scene.shape.Polygon?>
```
Comment (benhylak) [inline src/View/User/UserMapView.fxml:28]: can you attach a screenshot so I can see this without trying it out for now?

**Review by benhylak — COMMENTED** [review]
Please address these ASAP

File: src/Controller/User/UserMapViewController.java (hunk 1 above)
Comment (iancj97) [reply src/Controller/User/UserMapViewController.java]: Why?

File: src/View/User/UserMapView.fxml (hunk 4 above)
Comment (iancj97) [reply src/View/User/UserMapView.fxml:28]:
```
 ^
< >
```

Sorta like that.

Comment (iancj97) [reply src/View/User/UserMapView.fxml:28]: untitled

Comment (iancj97) [reply src/View/User/UserMapView.fxml:28]: @bhylak

Comment (benhylak) [reply src/View/User/UserMapView.fxml:28]: Put admin icon on top of the arrow changers

Comment (benhylak) [reply src/View/User/UserMapView.fxml:28]: Also can you make the arrows just a tad transparent and spaced a little more from the combo box

Comment (benhylak) [reply src/View/User/UserMapView.fxml:28]: perhaps a home icon too? So you can return to the home floor

**Comment by pjpolley** [comment]
displayforben

**Comment by benhylak** [comment]
1. Move the admin icon to the right. so its centered. 
2. Make the arrows the same color blue as the other buttons in UI 
3. Make them a little bit opaque (.65 ish)
4. Make combobox a lot less opaque (.9 ish)
5. Space out the combo box arrows so theres some padding 
6. Make the arrows a little wider (maybe a little taller) 
7. Move the combo box + arrows down a good bit

**Comment by pjpolley** [comment]
displayforben

**Review by benhylak — APPROVED** [review]
Looks better

**Pull request merged**